oomph-vscode/.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Set default behavior to automatically normalize line endings.
+* text=auto
+
```

Comment:
Is this really necessary?

Reply:
I don't know. The project skeleton for `oomph-vscode` is generated by [`yo`](https://www.npmjs.com/package/yo)

Reply:
[Here's what it does.(https://stackoverflow.com/questions/21472971/what-is-the-purpose-of-text-auto-in-gitattributes-file) I don't think this is necessary in Oomph, because you don't seem to have line ending problems with other files that don't use `.gitattributes`.